test/testsuites/csctargetnamespacetests.go

```diff
@@ -138,3 +150,74 @@ func succeededStateAfterTargetNamespaceCreated(t *testing.T) {
 		t.Fatalf("CatalogSourceConfig never reached expected phase/message, expected %v", expectedPhase)
 	}
 }
+
+// childResourcesCreated checks that once a CatalogSourceConfig is created that all expected runtime
+// objects are created as well.
+func childResourcesCreated(t *testing.T) {
+	// Get test namespace.
+	namespace, err := test.NewTestCtx(t).GetNamespace()
+	if err != nil {
+		t.Fatalf("Could not get namespace: %v", err)
+	}
+
+	// Check for the CatalogSource.
```

Comment:
These checks are being done during `OperatorSource` creation tests to. Please modularize.

Reply:
+1